src/components/OrderBook/index.tsx

```diff
@@ -1,12 +1,25 @@
 import {connect} from '../connect';
 import OrderBook from './OrderBook';
 
+interface OrderBookInterface {
```

Comment:
`OrderBookProps`

Reply:
plus you can write export declaration here like `export interface MyAwesomeProps`